x-pack/plugins/security_solution/scripts/endpoint/resolver_generator_script.ts

```diff
@@ -333,6 +374,29 @@ async function main() {
     );
   }
 
+  if (argv.rbacUser) {
+    const newRoleData = argv.rbacUser;
+    const role = await addRole(kbnClient, newRoleData ? { roleName: newRoleData } : undefined);
+    if (role) {
+      console.log(`Successfully added role: ${newRoleData}`);
+      const basicUser = await addUser(
+        client,
+        {
+          username: 'withHostIsolation',
+          password: 'changeme',
+        },
+        [newRoleData]
+      );
+      if (basicUser) {
+        console.log('Successfully created basic user');
+      } else {
+        console.log(`Failed to create user `);
+      }
+    } else {
+      console.log(`Failed to add role, ${newRoleData}`);
```

Comment:
Should this throw instead?

Reply:
the `addRole` function wraps the api call in a try / catch,  so the error should be caught. I can also make this else-statement `throw` but would that that be double kill?

Reply:
Just wondering if it should throw an error instead of `console.log`. Yes, the error will be handled by the try/catch, but if you don't throw anything here, the execution will continue with just a log. If this is what we are looking for, then yes, but if we wan't to stop the execution because the error, then I would suggest throwing an error instead.

Reply:
i think it makes sense to continue the execution if just the role/user making failed, especially if the user is also trying to create endpoints.